src/Compilers/CSharp/Test/IOperation/IOperation/IOperationTests_IObjectCreationExpression.cs

```diff
@@ -14600,74 +14600,46 @@ class A
     Statements (0)
     Next (Regular) Block[B1]
         Entering: {R1}
-
 .locals {R1}
 {
     Locals: [dynamic x]
     Block[B1] - Block
         Predecessors: [B0]
         Statements (1)
             ISimpleAssignmentOperation (OperationKind.SimpleAssignment, Type: dynamic, IsImplicit) (Syntax: 'x = 1')
-              Left: 
+              Left:
                 ILocalReferenceOperation: x (IsDeclaration: True) (OperationKind.LocalReference, Type: dynamic, IsImplicit) (Syntax: 'x = 1')
-              Right: 
+              Right:
                 IConversionOperation (TryCast: False, Unchecked) (OperationKind.Conversion, Type: dynamic, IsImplicit) (Syntax: '1')
                   Conversion: CommonConversion (Exists: True, IsIdentity: False, IsNumeric: False, IsReference: False, IsUserDefined: False) (MethodSymbol: null)
                     (Boxing)
-                  Operand: 
+                  Operand:
                     ILiteralOperation (OperationKind.Literal, Type: System.Int32, Constant: 1) (Syntax: '1')
-
         Next (Regular) Block[B2]
             Entering: {R2}
-
     .locals {R2}
     {
         CaptureIds: [0]
         Block[B2] - Block
             Predecessors: [B1]
-            Statements (1)
+            Statements (4)
                 IFlowCaptureOperation: 0 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'new A {[y:  ...  x] = { } }')
-                  Value: 
+                  Value:
                     IObjectCreationOperation (Constructor: A..ctor()) (OperationKind.ObjectCreation, Type: A) (Syntax: 'new A {[y:  ...  x] = { } }')
                       Arguments(0)
-                      Initializer: 
+                      Initializer:
                         null
-
-            Next (Regular) Block[B3]
-                Entering: {R3}
-
-        .locals {R3}
-        {
-            CaptureIds: [1] [2]
-            Block[B3] - Block
-                Predecessors: [B2]
-                Statements (2)
-                    IFlowCaptureOperation: 1 (OperationKind.FlowCapture, Type: null, IsImplicit) (Syntax: 'x')
```

Comment:
>IFlowCaptureOperation [](http://example.com/codeflow?start=20&length=21)

The graph validation didn't complain about the fact that the eliminated captures weren't used. We probably have a targeted exception in the validation logic. We should consider removing it if the exception no longer necessary. Consider at least opening a follow up issue.